x-pack/plugins/security_solution/public/management/pages/blocklist/view/blocklist.tsx

```diff
@@ -39,7 +39,7 @@ const BLOCKLIST_PAGE_LABELS: ArtifactListPageProps['labels'] = {
     defaultMessage: 'Blocklist',
   }),
   pageAboutInfo: i18n.translate('xpack.securitySolution.blocklist.pageAboutInfo', {
-    defaultMessage: '(DEV: temporarily using isolation exception api)', // FIXME: need wording from PM
+    defaultMessage: 'Add a blocklist to block applications or files from running.',
```

Comment:
Maybe add `endpoint` to the end of this sentence?

```suggestion
    defaultMessage: 'Add a blocklist to block applications or files from running on the endpoint.',
```